docs/faq.md

```diff
@@ -30,3 +30,51 @@ of dependencies and functionality that we do not need. At present we are using t
 `init` process, and a small set of minimal scripts, but we expect to replace that with a small
 standalone `init` process and a small piece of code to bring up the system containers where the
 real work takes place.
+
+## Console not displaying init or containerd output at boot
+
+If you're not seeing `containerd` logs in the console during boot, make sure that your kernel `cmdline` configuration doesn't list multiple consoles.
+
+`init` and other processes like `containerd` will use the last defined console in the kernel `cmdline`. When using `qemu`, to see the console you need to list `ttyS0` as the last console to properly see the output.
+
+## Troubleshooting containers
+
+Linuxkit runs all services in a specific namespace `services.linuxkit`. When entering the console, the logged in user is in the `getty` namespace.
```

Comment:
The word "namespace" is a bit overloaded and confusing here There are `containerd` namespaces (like `services.linuxkit`) and there are Linux namespace (`getty` runs in one). Maybe rephrase:

"LinuxKit runs all services in a specific `containerd` namespace called `services.linuxkit`. To list all defined containers use:"

Reply:
@rn, thanks, that's indeed better.
I just pushed a fix.